src/HindBrain/HindBrain.ino

```diff
@@ -52,17 +58,14 @@ ros::Publisher hitchPose("/hitch_pose", &currHitchPose);
 
 
 void setup() {
-
   // Initialize estop and auto-light switch
   eStop = new Estop(&nh, ESTOP_SENSE_PIN, ESTOP_DEBOUNCE_TIME);
   eStop->onStop(eStopTractor);
   eStop->offStop(eStartTractor);
   pinMode(ESTOP_RELAY_PIN, OUTPUT);
+  digitalWrite(ESTOP_RELAY_PIN, HIGH); // Stop tractor for safety
```

Comment:
This is the only part that I'm curious of its functionality. I'm scared that it will have not be able to turn itself to high or something like that. Maybe you can reason it out to me on Tuesday.